WordPress/Tests/DB/PreparedSQLPlaceholdersUnitTest.inc

```diff
@@ -207,3 +207,24 @@ $wpdb->query( $wpdb->prepare( "DELETE FROM $wpdb->wp_options WHERE option_name L
 $where .= $wpdb->prepare( " AND `name` LIKE '%%%%" . '%s' . "%%%%' ", $args['name'] ); // Bad x 2.
 
 $wpdb->query($wpdb->prepare("delete from wp_postmeta where post_id = $target_postId AND meta_key like 'google_snippets'")); // Bad, the PreparedSQL sniff will also kick in and throw an error about `$target_postId`.
+
+$field = 'post_type';
+$value = 'page';
+
+$wpdb->prepare( 'SELECT ID FROM %i WHERE %i = %s', $wpdb->posts, $field, $value ); // OK.
```

Comment:
Please also add an "OK" test with `i` placeholders with various different supported modifiers.

I realize that, in practice, it is unlikely that any modifier other than the `[argnum$]` will ever be used, but the sniff should still recognized and handle an `i` placeholder with other modifiers correctly.

Reply:
Yep, done :-)

Reply:
Thanks for that!